.cypress/cypress/integration/northamptonshire.js

```diff
@@ -0,0 +1,73 @@
+/*
+ * These don't run by default, need to run the tests with
+ * ./bin/browser-tests run --cobrand northamptonshire --area_id 2234 --coords '52.236251,0.892052
+ *
+ * NB: most of the non northamptonshire tests will then fail
```

Comment:
You can run e.g. `bin/browser-tests run ... --spec .cypress/cypress/integration/northamptonshire.js` to only run the Northamptonshire file.

Reply:
Although, shouldn't the `only_or_skip` below be suppressing all other tests in this instance?

Reply:
I guess that is only per-file or something and I didn't realise. Could add similar to the other test files I guess easily enough.
Or better, think of a way of having them all run - I assume I couldn't find a way to change the host name :-/